src/components/organisms/UiMultipleChoices/UiMultipleChoices.vue

```diff
@@ -146,27 +145,10 @@ const updateHandler = (newValue: MultipleChoicesModelValue, index: number) => {
 };
 
 function focusInvalidChoice() {
-  if (invalidInputsRefs.size > 0) {
-    const element = [ ...invalidInputsRefs.values() ][0];
-    focusElement(element, true);
-  }
+  const firstInvalidChoice = multipleChoicesItemRefs.value.find((element, index) => hasError(index));
```

Comment:
WDYT about `computed` for `firstInvalidChoices`? `computed` is cashable and will be alway updated.

Reply:
This was the reason I created this PR though. If it does not need to be always updated, why do it? We only use it on focus, so it fits better to calculate it only on focus.